modules/cnv-monitoring-vm-migration-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-monitor-vmi-migration.adoc
+
+[id="cnv-monitoring-vm-migration-web_{context}"]
+= Monitoring live migration of a virtual machine instance in the web console
+
+For the duration of the migration, the virtual machine will have a status of 
+*Migrating*. This is displayed in the *Virtual Machines* list, or in the 
+*Virtual Machine Details* screen for the migrating virtual machine. 
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
```

Comment:
Same as before, I'd pick one, but if you keep them both, please reformat the steps.

Since there's one step, lead with * instead of .